src/builder/edit_role.rs

```diff
@@ -177,7 +177,9 @@ impl<'a> EditRole<'a> {
         };
 
         if let Some(position) = self.position {
-            http.edit_role_position(guild_id, role.id, position, self.audit_log_reason).await?;
+            guild_id
+                .edit_role_position_with_reason(http, role.id, position, self.audit_log_reason)
+                .await?;
```

Comment:
This introduced a dependency on the `model` feature inside code that is normally just gated behind `http` plus `builder`. Is it possible to fix this?

Reply:
I'm going to make a PR soon that moves `builder` inside of `model`, since `builder` shouldn't be used without `model` anyway.

Reply:
Can you elaborate? I'm thinking of the builder-no-http usecase as a specific counter-example to that idea.

Reply:
Let's move this to the lib-development chat as this is going quite off-topic for this PR.